### PR TITLE
Exercise5

### PR DESCRIPTION
**Pull request by clucksupremacy**
A lot of the code is repeats with different variables. Is there a way to make it more concise?

### CODE REVIEW

**Comment by clucksupremacy** [comment]
i am so dumb

**Review by Aceeri — APPROVED** [review]
Goob! now separate it out into functions :)

File: exercise-5.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+function crypto_calculator(money, bitcoin_price, ethereum_price, litecoin_price) {
+    if (money <= 0) {
+        console.log("Insufficient funds");
+        return [0, 0, 0];
+    }
+
+    let bitcoin_max = 0;
+    let ethereum_max = 0;
+    let litecoin_max = 0;
```
Comment (Aceeri) [inline exercise-5.js]: Try consolidating these with the assignments

File: exercise-5.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+function crypto_calculator(money, bitcoin_price, ethereum_price, litecoin_price) {
+    if (money <= 0) {
+        console.log("Insufficient funds");
+        return [0, 0, 0];
+    }
+
+    let bitcoin_max, ethereum_max, litecoin_max = 0;
```
Comment (Aceeri) [inline exercise-5.js]: Good job on the consolidating similar value variables I hadn't thought about that, but by the consolidating I meant removing these lines entirely.

File: exercise-5.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    function coin_max(price, max) {
+        if (price < 0) {
+            max = undefined;
+        } else if (price == 0) {
+            max = 0;
+        } else {
+            max = Math.floor(money / price);
+        }
+        return max; // returns the value but doesn't store it in a variable. Make sure to do that yourself
+    }
```
Comment (Aceeri) [inline exercise-5.js]: For example, try multiple returns here

File: exercise-5.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    bitcoin_max = coin_max(bitcoin_price, bitcoin_max);
+    ethereum_max = coin_max(ethereum_price, ethereum_max);
+    litecoin_max = coin_max(litecoin_price, litecoin_max);
```
Comment (Aceeri) [inline exercise-5.js]: and try creating the variables here :)

File: exercise-5.js (hunk 3 above)
Comment (clucksupremacy) [reply exercise-5.js]: Same return statement multiple times?

Comment (Aceeri) [reply exercise-5.js]: no no, in each of the conditions have a return statement

Comment (clucksupremacy) [reply exercise-5.js]: in each condition have the same return statement?

Comment (clucksupremacy) [reply exercise-5.js]: i got it

**Review by Aceeri — APPROVED** [review]
goob!

**Pull request merged**